asdf/tests/test_compression.py

```diff
@@ -90,3 +90,18 @@ def test_bzp2(tmpdir):
     tree = _get_large_tree()
 
     _roundtrip(tmpdir, tree, 'bzp2')
+
+
+def test_recompression(tmpdir):
+    tree = _get_large_tree()
+    tmpfile = os.path.join(str(tmpdir), 'test1.asdf')
+    file = asdf.AsdfFile(tree)
```

Comment:
Just a nit but it would be preferred if this variable had a name that does not shadow a builtin.